src/main/java/com/zeepy/server/community/repository/CommunityRepository.java

```diff
@@ -0,0 +1,7 @@
+package com.zeepy.server.community.repository;
+
+import com.zeepy.server.community.domain.Community;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface CommunityRepository extends JpaRepository<Community,Long> {
```

Comment:
```suggestion

@Repository
public interface CommunityRepository extends JpaRepository<Community,Long> {
```
Repository어노테이션을 달아주세요.